lib/Dialect/FIRRTL/Import/FIRTokenKinds.def

```diff
@@ -66,7 +66,9 @@ TOK_PUNCTUATION(question, "?")
 TOK_PUNCTUATION(l_paren, "(")
 TOK_PUNCTUATION(r_paren, ")")
 TOK_PUNCTUATION(l_brace, "{")
+TOK_PUNCTUATION(l_brace_bar, "{|")
 TOK_PUNCTUATION(r_brace, "}")
+TOK_PUNCTUATION(r_brace_bar, "{|")
```

Comment:
```suggestion
TOK_PUNCTUATION(r_brace_bar, "|}")
```

(I'm unsure how the test passes with this being wrong?)

re:naming scheme... should this be `r_brace_bar` to indicate it's the right-side "brace-bar" paren-like (pairs with `l_brace_bar`), or `bar_r_brace` to describe the token's contents?

Reply:
(Oh, because the "spelling" portion of these doesn't appear to be used anywhere :+1: .)